assessment/assessment_engine.go

```diff
@@ -75,7 +75,7 @@ func initializeCollectors(conv *internal.Conv, sourceProfile profiles.SourceProf
 		return c, err
 	}
 	c.sampleCollector = &sampleCollector
-	infoSchemaCollector, err := assessment.CreateInfoSchemaCollector(conv, sourceProfile)
+	infoSchemaCollector, err := assessment.CreateNewInfoSchemaCollector(conv, sourceProfile)
```

Comment:
nit: creating will return a new object by definition. I'm wondering if the renaming is required.